database/database.go

```diff
@@ -0,0 +1,41 @@
+package database
+
+import (
+	"database/sql"
+	"os"
+)
+
+func ConnectDB(dbFile string) (*sql.DB, error) {
+	db, err := sql.Open("sqlite3", dbFile)
+	if err != nil {
+		return nil, err
```

Comment:
Ошибки лучше оборачивать, чтобы понимать откуда они пришли
```suggestion
		return nil, fmt.Sprintf("error while openning %s: %w", dbFile, err)
```

Reply:
Это лучше делать везде, чтобы отследить цепочку и дополнить ошибку доп информацией, например, как тут именем файла